lib/cli.js

```diff
@@ -137,6 +137,18 @@ const cli = {
 
             log.info("v" + require("../package.json").version);
 
+        } else if (currentOptions.printConfig) {
```

Comment:
What happens if I do this:

```
$ eslint --print-config foo.js bar.js
```

In our current implementation, we get a warning. If I'm reading this code correctly, the new version would print the config for foo.js and not warn at all about bar.js, which I think is misleading. We probably want to check that `files.length` is zero, and if not, throw up the same warning we currently do.